test/03-simple-add-new-form-test.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+import SimpleAddNewForm from '../src/03/SimpleAddNewForm';
+
+const stub = () => {};
+
+const setup = (customProps = {}) => {
+  const props = Object.assign({}, {
+    title: 'Add new song',
+    inputFieldPlaceholder: 'Enter your favorite song...',
+    handleSubmit: stub
+  }, customProps);
+
+  return mount( <SimpleAddNewForm {...props} />);
+}
+
+describe('Component: SimpleAddNewForm', () => {
+
+  xit('render a div with class `add-new-form`', () => {
+    const component = setup();
+
+    expect(component.find('div.add-new-form')).to.have.length(1);
+  });
+
+  xit('renders the title', ()=> {
+    const component = setup();
+
+    const title = component.props().title;
+    expect(component.text()).to.include(title);
+  });
+
+  xit('renders an input field and a button', ()=> {
+    const component = setup();
+
+    expect(component.find('input')).to.have.length(1);
+    expect(component.find('button')).to.have.length(1);
+  });
+
+  describe('when submitted', () => {
+
+    describe('and the input field is empty', ()=> {
+
+      xit('there\'s a placeholder in the input field', ()=> {
+        const component = setup();
+
+        const placeholder = component.props().inputFieldPlaceholder;
+        expect(component.find('input').props().placeholder).to.equal(placeholder);
+      });
+
+      xit('the button is disabled', ()=> {
+        const component = setup();
+
+        expect(component.find('button').html()).to.equal(`<button disabled="">${component.props().title}</button>`);
```

Comment:
I have validated in the browser that the button is disabled when the input field is empty, but I expected the html to be `disabled=true` instead of `disabled=""`. Let me know if you know of a better way to test this.